forest/src/logger/mod.rs

```diff
@@ -10,20 +10,9 @@ pub(crate) fn setup_logger(log_config: &LogConfig) {
     let mut logger_builder = pretty_env_logger::formatted_timed_builder();
 
     // Assign default log level settings
-    logger_builder.filter(Some("libp2p_gossipsub"), LevelFilter::Error);
-    logger_builder.filter(Some("filecoin_proofs"), LevelFilter::Warn);
-    logger_builder.filter(Some("storage_proofs_core"), LevelFilter::Warn);
-    logger_builder.filter(Some("surf::middleware"), LevelFilter::Warn);
-    logger_builder.filter(
-        Some("bellperson::groth16::aggregate::verify"),
-        LevelFilter::Warn,
-    );
-    logger_builder.filter(Some("tide"), LevelFilter::Warn);
-    logger_builder.filter(Some("libp2p_bitswap"), LevelFilter::Warn);
-    logger_builder.filter(Some("rpc"), LevelFilter::Info);
     logger_builder.filter(None, LevelFilter::Info);
 
-    for item in &log_config.log_values {
+    for item in &log_config.0 {
```

Comment:
It's tempting to change the argument of `setup_logger` to `&[LogValue]`. Since `LogConfig` is just a type synonym, we can add a `Deref` impl to do the conversation automatically.

Reply:
Hubert did this.